code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -400,7 +400,7 @@ var/const/PATCH = 4 //patches
 
 /datum/reagents/proc/check_gofast(mob/M)
 	if(istype(M, /mob))
-		if(M.reagents.has_reagent("unholywater")||M.reagents.has_reagent("nuka_cola"))
+		if(M.reagents.has_reagent("unholywater")||M.reagents.has_reagent("nuka_cola")||M.reagents.has_reagent("stimulants"))

```

Comment:
I'd rather saving the refactoring to someone else to be honest; I just wanted to get stimulants in a functioning status.